src/array.js

```diff
@@ -164,7 +166,7 @@ inherits(ArraySchema, MixedSchema, {
 
   ensure() {
     return this.default(() => []).transform(
-      val => (val === null ? [] : [].concat(val)),
+      val => (val === null ? [] : castArray(val)),
```

Comment:
lets avoicd the lodash function and go with
```
if (this.isType(val)) return val
return val === null ? [] : [].concat(val)
```

Reply:
Oops…
Done.